packages/eslint-plugin/docs/rules/no-confusing-void-expression.mdx

```diff
@@ -119,6 +119,24 @@ function doSomething() {
 console.log(void alert('Hello, world!'));
 ```
 
+### `ignoreVoidReturningFunctions`
+
+Whether to ignore returns from functions with explicit `void` return types and functions with contextual `void` return types.
```

Comment:
[Docs] I think most users won't understand what _"contextual"_  means. How does this strike you?

```suggestion
Whether to ignore returns from functions with `void` return types when inside a function with a `void` return type.
```

Reply:
Yes! This is a lot better, thanks ❤️